src/system/Card/Card.tsx

```diff
@@ -1,39 +1,64 @@
+/** @jsxImportSource theme-ui */
+
 /**
  * External dependencies
  */
 import classNames, { Argument } from 'classnames';
 import { forwardRef, Ref } from 'react';
-import { BoxProps } from 'theme-ui';
+import { BoxProps, useThemeUI, get } from 'theme-ui';
 
 /**
  * Internal dependencies
  */
 import { Box } from '..';
 
+export enum CardVariant {
+	'primary',
+	'secondary',
+	'indent',
+}
+
 export interface CardProps {
-	variant?: string;
+	variant?: keyof typeof CardVariant;
 	sx?: BoxProps[ 'sx' ];
 	className?: Argument;
+	header?: string;
+	children?: React.ReactNode;
 }
 
 type CardBoxProps = CardProps & BoxProps;
 
 export const Card = forwardRef< HTMLElement, CardBoxProps >(
 	(
-		{ variant = 'primary', sx = {}, className, ...props }: CardBoxProps,
+		{ variant = 'primary', header, sx = {}, className, children, ...props }: CardProps,
 		ref: Ref< HTMLElement >
-	) => (
-		<Box
-			ref={ ref }
-			sx={ {
-				// pass variant prop to sx
-				variant: `cards.${ variant }`,
-				...sx,
-			} }
-			className={ classNames( 'vip-card-component', className ) }
-			{ ...props }
-		/>
-	)
+	) => {
+		const { theme } = useThemeUI();
+		const cardVariant = get( theme, `cards.${ variant }`, {} );
+		const basePadding = cardVariant?.p || cardVariant?.padding || 3;
```

Comment:
This seems odd to me. Can you add this default padding to the primary variant and let the variants decide the `p` style inside the card body?

It does not feel "correct" to use the card body padding inside of the `p` tag; this can lead to unexpected side effects when debugging an issue. Does it make sense?

My suggestion:

- Always use padding coming from the variant into the Box
- Let the variant style the `p`